src/DevL10N/xml/1.2/Languages/Japanese/Keyed/AAA.xml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Whoops, pushed this file by mistake. I already deleted it on upstream before I released 1.3.0, sorry!

Reply:
OK!